### ISSUE TITLE
incompatible with 'python=3.8'

### ISSUE THREAD

**Issue by AnkS4** [facebook/prophet#1640]
I tried to install fbprophet using conda. I'm getting following error:

```
$ conda install -c conda-forge fbprophet
Collecting package metadata (current_repodata.json): done
Solving environment: failed with initial frozen solve. Retrying with flexible solve.
Solving environment: failed with repodata from current_repodata.json, will retry with next repodata source.
Collecting package metadata (repodata.json): done
Solving environment: failed with initial frozen solve. Retrying with flexible solve.
Solving environment: / 
Found conflicts! Looking for incompatible packages.
This can take several minutes.  Press CTRL-C to abort.
failed                                                                                                                               

UnsatisfiableError: The following specifications were found
to be incompatible with the existing python installation in your environment:

Specifications:

  - fbprophet -> python[version='2.7.*|3.5.*|3.6.*|>=2.7,<2.8.0a0|>=3.6,<3.7.0a0|>=3.7,<3.8.0a0|>=3.5,<3.6.0a0']

Your python: python=3.8

If python is on the left-most side of the chain, that's the version you've asked for.
When python appears to the right, that indicates that the thing on the left is somehow
not available for the python version you are constrained to. Note that conda will not
change your python version to a different minor version unless you explicitly specify
that.
```

But, using pip it gets installed without any warnings, python version: 3.8.5-1.

**Comment by bletham** [comment]
The issue is in conda-forge. The build for python38 is failing, and I have spent some effort to try and fix it but so far have failed. I'm really confused as to why it is failing when the other py versions have built fine, I think it will require someone with more conda experience than me. The failing build is right here: https://github.com/conda-forge/fbprophet-feedstock/pull/31 . This is related to #1227.

**Comment by juniorspqr** [comment]
I have receive the same issue AnkS4, and I have just download python 3.7.7 and it's work 
As you can see on the error "Specification" > 3.7 but < 3.8 so you cannot take a Python version = or > at 3.8 ... 

ps : don't forget to manage your env path to  anaconda to use this new python version.

**Comment by bletham** [comment]
This should be fixed in conda forge, with the new version 0.7.1. If there are still issues with this, go ahead and re-open #1227. Thanks!
